### PR TITLE
Unify Vega, Voila, quilt_summarize, height docs (and crush images)

### PR DESCRIPTION
**Pull request by akarve**
<del>
- [ ] Unit tests
- [ ] Automated tests (e.g. Preflight)
- [ ] Documentation
    - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [ ] JavaScript: basic explanation and screenshot of new features
- [ ] [Changelog](CHANGELOG.md) entry
</del>

### CODE REVIEW

File: docs/Catalog/VisualizationDashboards.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    }
+  ]
+}
+```
+
+![](../imgs/catalog_echarts.png)
+
+### External files
+
+As with Vega, you can you can provide either a relative path or URL to the dataset file.
```
Comment (fiskus) [inline docs/Catalog/VisualizationDashboards.md:183]:
```suggestion
As with Vega, you can provide either a relative path or URL to the dataset file.
```

**Comment by fiskus** [comment]
@akarve don't know what was happened with the images. Probably, you modified them from `pn24` to `png8`? I see that colors were changed, nothing more:
https://github.com/quiltdata/quilt/blob/master/docs/imgs/admin-bucket.png
vs
https://github.com/quiltdata/quilt/blob/visdocs/docs/imgs/admin-bucket.png

File: docs/Catalog/VisualizationDashboards.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+Each element of an array in `quilt_summarize.json` can either be a path string
+or an object with one or more of the following properties:
+
+- `path` - file path relative to `quilt_summarize.json`
+- `title` - title rendered instead of file path
+- `description` - description in markdown format
+- `width` - column width either in pixels or ratio (default is ratio `1`)
+- `types` - a list of rendering types; currently only singleton list values are supported:
+    - `["echarts"]` to render JSON as an EChart
+    - `["voila"]` to render a Jupyter notebooks as an interactive Voila dashboard
```
Comment (fiskus) [inline docs/Catalog/VisualizationDashboards.md:69]:
```suggestion
    - `["voila"]` to render a Jupyter notebook as an interactive Voila dashboard
```

File: docs/Catalog/VisualizationDashboards.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+Enterprise deployments of Quilt support interactive Jupyter notebooks
+with [Voilà](https://github.com/voila-dashboards/voila).
+
+In brief, a Voila dashboard version of your notebook will
+display all of the output cells and none of the input cells from the underlying
+notebook. This enables you to create interactive, Jupyter-driven apps for your
+Quilt catalog users.
+
+The Voila libraries execute a remote Jupyter Kernel and stream the results to the
+browser with Tornado. Jupyter kernels run on single EC2 instance (`t3.small` by default)
```
Comment (fiskus) [inline docs/Catalog/VisualizationDashboards.md:226]:
```suggestion
browser with Tornado. Jupyter kernels run on a single EC2 instance (`t3.small` by default)
```

File: docs/Catalog/VisualizationDashboards.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+```json
+// quilt_summarize.json
+[
+  "file1.json",
+  "file2.csv",
+  "file3.ipynb"
+]
+```
+
+By default each list element renders in its own row.
```
Comment (fiskus) [inline docs/Catalog/VisualizationDashboards.md:37]:
```suggestion
By default, each list element renders in its own row.
```

**Review by fiskus — APPROVED** [review]

**Comment by nl0** [comment]
> @akarve don't know what was happened with the images. Probably, you modified them from `pn24` to `png8`? I see that colors were changed, nothing more: https://github.com/quiltdata/quilt/blob/master/docs/imgs/admin-bucket.png vs https://github.com/quiltdata/quilt/blob/visdocs/docs/imgs/admin-bucket.png

yeah they have been crushed

**Comment by fiskus** [comment]
My point was that I aware of image size changes, but since I don't know what exactly was a cause, I don't know what to review in these changes. Or "crushing images" is the specific term?
I reviewed a couple of images and see that colors were changed, maybe, it's an unwanted change

**Comment by akarve** [comment]
> My point was that I aware of image size changes, but since I don't know what exactly was a cause, I don't know what to review in these changes. Or "crushing images" is the specific term? I reviewed a couple of images and see that colors were changed, maybe, it's an unwanted change

Thanks for keeping an eye on this. Color changes are OK. I use ImageOptim and it probably does PNG8 or some RLE encoding improvements. Since these are in repo I like the small but big enough to look good on open.

**Comment by sir-sigurd** [comment]
For me "crush" sounds like images were optimized with [pngcrush](https://en.wikipedia.org/wiki/Pngcrush) which does only lossless optimization.

**Pull request merged**